pom.xml

```diff
@@ -278,7 +278,7 @@
 			<dependency>
 				<groupId>org.liquibase</groupId>
 				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
+				<version>3.5.0</version>
```

Comment:
are you upgrading to the latest liquibase core or you need 3.5.x?

Reply:
> are you upgrading to the latest liquibase core or you need 3.5.x?

that is a huge upgrade that can't just work out of the box so i am trying to go down version by version till i get one that compiles successfully